From: Simon Vogl <simon.vogl@gmx.net>
Date: Fri,  5 Mar 2021 17:54:41 UTC
Subject: [PATCH] Fix unintended low quality of YouTube videos inside QMPlay2.

This patch is required to prevent QMPlay2 from selecting wrong quality presets for YouTube.

diff --git a/src/modules/Extensions/YouTube.cpp b/src/modules/Extensions/YouTube.cpp
index 0ffbfb5c..78ee6755 100644
--- a/src/modules/Extensions/YouTube.cpp
+++ b/src/modules/Extensions/YouTube.cpp
@@ -961,10 +961,10 @@ QStringList YouTube::getYouTubeVideo(const QString &param, const QString &url, I
         if (format.isEmpty())
             continue;
 
-        const auto container = format["container"].toString();
-        if (container.contains("dash", Qt::CaseInsensitive))
+        const auto protocol = format["protocol"].toString();
+        if (protocol.contains("dash", Qt::CaseInsensitive))
         {
-            // Skip MP4 DASH, because it doesn't work properly
+            // Skip MP4 DASH, because it doesn't work properly (old comment, is it still valid?)
             continue;
         }
 
